chia_puzzles_py/programs.py

```diff
@@ -331,6 +337,136 @@
 SINGLETON_TOP_LAYER_HASH = bytes.fromhex(
     "24e044101e57b3d8c908b8a38ad57848afd29d3eecc439dba45f4412df4954fd"
 )
+VAULT_1_OF_N = bytes.fromhex(
```

Comment:
While we developed these puzzles in the context of the vault, the standard is meant to be widely applicable and I don't think we should have commitments to the vaulting primitive in the name.

Reply:
Renamed to ONE_OF_N, N_OF_N, and so on